src/code/views/teacher-view.tsx

```diff
@@ -136,6 +143,21 @@ export class TeacherView extends React.Component<
     }
   }
 
+  handleCellClick = (cell:IGridCell, evt: React.MouseEvent<HTMLElement>) => {
+    // Prevent reopening if we're in the process of closing.
+    if (this.closingCount) { return; }
+    const classes = evt.currentTarget.className;
+    const match = /grid-cell-label-([A-Z]-[0-9]*)/.exec(classes);
+    const cellName = match && match[1] || null;
```

Comment:
Couldn't you use the `displayName` method on the cell?   Grepping it from the CSS seems like it might cause trouble in the future.

Reply:
Funny you should mention that. I wrote that code originally when the only argument to the function was the event, and so the cell had to be derived from the DOM element. Then I modified the callback so that the grid cell would be passed as well, but never actually used the grid cell to eliminate the need to extract it from the DOM element. Good catch. Fixed in a subsequent commit.